### PR TITLE
setup codecov for airlfow microservice

### PR DESCRIPTION
**Pull request by Mnoble-19**
**_WHAT DOES THIS PR DO?_**
* Fixes previous errors causing `code_coverage` to fail
* Setups Code Coverage for airflow microservice
* Cleans up file structure and messy tests in the tests folder

Screenshot 2023-10-11 at 17 32 19

### CODE REVIEW

File: src/airflow/airqo_etl_utils/config.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ class Config:
     MLFLOW_TRACKING_URI = os.getenv("MLFLOW_TRACKING_URI")
     FORECAST_MODELS_BUCKET = os.getenv("FORECAST_MODELS_BUCKET")
     MONGO_URI = os.getenv("MONGO_URI")
-    MONGO_DATABASE_NAME = os.getenv("MONGO_DATABASE_NAME")
+    MONGO_DATABASE_NAME = os.getenv("MONGO_DATABASE_NAME", "airqo_db")
```
Comment (BenjaminSsempala) [inline src/airflow/airqo_etl_utils/config.py:173]: Just a reminder to add this to the .env after this has been merged
@Mnoble-19 
@Baalmart

Comment (Mnoble-19) [reply src/airflow/airqo_etl_utils/config.py:173]: Thanks @BenjaminSsempala , let me actually just remove it, and use an alternative approach

**Comment by Baalmart** [comment]
Hi @Mnoble-19 , please review the merge conflicts in this PR. cc: @Psalmz777

**Comment by Mnoble-19** [comment]
> Hi @Mnoble-19 , please review the merge conflicts in this PR. cc: @Psalmz777

Done @Baalmart

**Pull request merged**